lib/transpilation/nunjucks_transpiler.js

```diff
@@ -5,4 +5,21 @@
 
 const assetPath = (asset, version) => `{{ asset_path }}${asset}?${version}`
 
+const kebabToCamel = string => {
+  return string.replace(/-([a-z])/g, function (m, w) {
+    return w.toUpperCase()
+  })
+}
```

Comment:
i like this library for it, helps with css to js etc too 
https://github.com/blakeembrey/change-case